README.md

```diff
@@ -184,6 +184,18 @@ In some situations, like for Protocol Buffer schemas, we want to generate second
 }
 ```
 
+### Heuristics
+
+Wildcard imports in Java and Scala can be expensive to resolve, since every subsequent import might be relative to the previous wildcard.
```

Comment:
Java imports are always rooted. (This comes up when scalac reads Java sources.) Sorry I don't know the context, so my comment may be irrelevant.